taqueria-plugin-metadata/index.ts

```diff
@@ -0,0 +1,34 @@
+import { Option, Plugin, PositionalArg, Task } from '@taqueria/node-sdk';
+import proxy from './src/proxy';
+
+Plugin.create(() => ({
+	schema: '0.1',
+	version: '0.4.0',
+	alias: 'metadata',
+	tasks: [
+		Task.create({
+			task: 'metadata',
+			command: 'generate metadata [contractName]',
+			description: 'Create contract metadata.',
+			aliases: [],
+			handler: 'proxy',
+			positionals: [
+				PositionalArg.create({
+					placeholder: 'contractName',
+					description: 'Which contract?',
+					type: 'string',
+				}),
+			],
+			encoding: 'none',
+		}),
+		Task.create({
+			task: 'project-metadata',
+			command: 'generate-project-metadata',
```

Comment:
Had to use `generate-project-metadata` to avoid a taq bug when multiple commands have the same first word

Reply:
To reproduce:
- Find replace `generate-project-metadata` with `generate project-metadata` (here and in test)
- Rebuild this project
- Run e2e test for this project
- **Result:** All tests will now fail

Reply:
Reproduced here: #1147